packages/validator/test/e2e/slashing-protection-interchange-tests/run.test.ts

```diff
@@ -12,92 +12,98 @@ import {
   InvalidAttestationError,
   InvalidBlockError,
 } from "../../../src/slashingProtection";
-import {downloadTests, loadTestCases as newLoadTestCases} from "./downloadTest";
+import {downloadTests, loadTestCases} from "./index.test";
```

Comment:
Importing from .test files in not very standard. Consider refactoring

Reply:
it didn't really need to be named a test file, so i just renamed it.